src/python/pants/engine/internals/rule_visitor.py

```diff
@@ -197,12 +197,12 @@ def _get_awaitable(self, call_node: ast.Call, is_effect: bool) -> AwaitableConst
         get_args = call_node.args
         parse_error = partial(GetParseError, get_args=get_args, source_file_name=self.source_file)
 
-        if len(get_args) not in (2, 3):
+        if len(get_args) < 1 or len(get_args) > 3:
```

Comment:
Just add the additional `1` would keep this more readable, imo.
```suggestion
        if len(get_args) not in (1, 2, 3):
```